### DISCUSSION TITLE
does anybody have any games that are or can be crammed into one html file?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1408] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
??

**Comment by Luigi-Master** [comment]
bro why in one html file?

**Reply by ghost** [reply]
So I can play games forever unblocked because there are thousands of ways to preview and run html.

**Comment by akabutnicer** [comment, marked as answer]
this is fairly similar to your objective: https://gitlab.com/3kh0/3kh0-assets
we cannot help you if otherwise, as it's quite unclear how to easily decide which games are one HTML file (plus most games aren't able to in the first place)

**Reply by ghost** [reply]
Thanks.
